article/_posts/2021-07-22-Perceiver.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: review
+title: "Perceiver: General Perception with Iterative Attention"
+tags: deep-learning attention transformer
+author: "Antoine Théberge"
+cite:
+    authors: "Andrew Jaegle, Felix Gimeno, Andrew Brock, Andrew Zisserman, Oriol Vinyals, Joao Carreira"
+    title:   "Perceiver: General Perception with Iterative Attention"
+    venue:   "arXiv"
+pdf: "https://arxiv.org/pdf/2103.03206.pdf"
+---
+
+
+# Highlights
+
+- New architecture that can handle multiple modalities without tailoring the architecture to each
+
+# Introduction
+
+> Biological systems perceive the world by simultaneously processing high-dimensional inputs from modalities as diverse as vision, audition, touch, proprioception, etc.
+
+Convolutional neural networks have demonstrated tremendous performances on vision tasks, but are virtually useless for other modalities. Transformers, on the other hand, are highly suited for text-based tasks, but scale poorly to vision tasks. Previous architecture focused on multiple modalities often had to make several choices on when and how to merge them. 
+
+In this work, the authors introduce the _Perceiver_ architecture, able to leverage arbitrary modalities by iteratively projecting each one to a latent space using attention mechanisms and transformers.
+
+
+# Methods
+
+![](/article/images/perceiver/fig1.jpeg)
+
+Figure 1. sums the method. "Byte array" represents arbitrary inputs unrolled as an array. The method iteratively uses two components to tame the input complexity and variety: cross-attention modules and transformers. Each modality is input one after the other. As such, the Perceiver architecture can be seen as an RNN where each input is not a timepoint but a new modality.
```

Comment:
sums up ?

Reply:
True, fixed in c25513de667ea3bef42c0cd2b7b6cb3a5c9d45f8